apps/src/codebridge/utils/dragAndDropUtils.ts

```diff
@@ -198,3 +197,36 @@ export const getHighestPriorityCollision = (
 
   return [rectangleCollisions[0]];
 };
+
+// Extend sortableKeyboardCoordinates to support tab and shift+tab to move items.
+export const sortableKeyboardCoordinatesWithTab: KeyboardCoordinateGetter = (
+  event,
+  args
+) => {
+  if (event.code === KeyboardCode.Tab && event.shiftKey) {
```

Comment:
I'm curious about the order of operations here considering users press shift first, but if it's working then I'm all for it!

Reply:
It will only work if the user is holding shift when they click tab. If they click shift, then tab it won't work. Is that ok? The arrow keys are a single keystroke for the same behavior.

Reply:
Perfect!